scoring/templates/scoring/council.html

```diff
@@ -226,77 +243,163 @@ <h3 class="exclamation text-white">Visit us again on a bigger screen</h3>
+                              {{ council.name }} was only awarded penalty marks in this section
+                            {% else %}
+                              {{ council.name }} incurred a penalty of {{ section.negative_percent|floatformat:0 }}% in this section
+                            {% endif %}
+                        </p>
+                        <p class="fw-normal mt-1 mb-0">
+                            {{ section.score }} &equals; Total score ({{ section.non_negative_max|floatformat:"-2" }}) &minus; Penalty points ({{ section.negative_points }})
+                        </p>
                     </div>
                     {% endif %}
 
-                    <button type="button" class="d-block mt-2 d-md-none btn btn-outline-primary btn-sm js-toggle-council-question-table-section rounded-0" aria-label="Expand this section" title="Expand this section">
+                    <button type="button" class="d-block mt-2 d-lg-none btn btn-outline-primary btn-sm js-toggle-council-question-table-section rounded-0" aria-label="Expand this section" title="Expand this section">
                         {% include 'caps/icons/chevron-down.html' with classes='align-text-bottom' width='1.2em' height='1.2em' role='presentation' %}
-                        Show questions
+                        Show questions list
                     </button>
                 </td>
 
               {% if section.top_performer %}
-                <td class="score border-bottom border-opacity-25 border-primary is--section-score">
+                <td  class="score border-bottom border-opacity-25 border-primary is--section-score display-only-large-up">
                     {% include 'caps/icons/scorecards-star.html' with classes='text-info align-text-bottom me-1' width='1.2em' height='1.2rem' role='presentation' %}
                     <span>{{ section.weighted_score|floatformat:0 }}%</span>
                 </td>
+
+                </td>
+                {% comment %} TODO: Include previous scorecards score(2023) {% endcomment %}
+                <td class="js-previous-year-score display-only-large-up">{{ section.weighted_score|floatformat:0 }}%</td>
+
+                <td class="display-only-large-up">
+                    {% comment %} TODO: IF Negative variation: Change bg-green-100 to bg-red-100 {% endcomment %}
+                    {% comment %} TODO: IF Negative variation: Change text-success to text-danger  {% endcomment %}
+                    {% comment %} TODO: IF Negative variation: Change &#8593; to &#8595; {% endcomment %}
+                    {% comment %} TODO: IF No variation: Change bg-green-100 to bg-gray-300 {% endcomment %}
+                    {% comment %} TODO: IF No variation: Change text-success to text-dark {% endcomment %}
+                    {% comment %} TODO: IF No variation: display '--' {% endcomment %}
+                    <span class="badge align-text-bottom bg-green-100 text-success text-center">&#8593; 5%</span>
```

Comment:
I wonder whether this built-in typographical arrow is just a bit weedy? It’s displayed quite small, and I worry a little bit that people might mistake it for a "1".

We could make a up/down triangle out of CSS borders if we wanted. Or include a simple, chunkier arrow as an SVG or a background image (and could even rotate it with CSS to save having two versions).

Reply:
Happy to add the SVG ones, I think it's simpler and follows the concept of what we have been doing.